lib/js/_publish/_publish-package.js

```diff
@@ -10,15 +10,23 @@ const reporter = require('@springernature/util-cli-reporter');
 // Publish command, to public registry
 let command = 'npm publish --access public';
```

Comment:
const?

Reply:
yep, should be. I think this used to get added to in a previous version, but i'll flip it to `const`.